pyproject.toml

```diff
@@ -0,0 +1,113 @@
+[project]
+name = "datacompy"
+description = "Dataframe comparison in Python"
+readme = "README.md"
+authors = [
+  { name="Ian Robertson" },
+  { name="Dan Coates" },
+  { name="Faisal Dosani", email="faisal.dosani@capitalone.com" },
+]
+maintainers = [
+  { name="Faisal Dosani", email="faisal.dosani@capitalone.com" }
+]
+license = {text = "Apache Software License"}
+dependencies = [
+    "pandas<=2.0.2,>=0.25.0",
+    "numpy<=1.24.3,>=1.22.0",
+    "ordered-set<=4.1.0,>=4.0.2",
+    "fugue<=0.8.5,>=0.8.5",
+]
+requires-python = ">=3.8.0"
+classifiers = [
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+]
+
+dynamic = ["version"]
+
+[project.urls]
+Homepage = "https://github.com/capitalone/datacompy"
+Documentation = "https://capitalone.github.io/datacompy/"
+Repository = "https://github.com/capitalone/datacompy.git"
+"Bug Tracker" = "https://github.com/capitalone/datacompy/issues"
+"Source Code" = "https://github.com/capitalone/datacompy"
+
+[tool.setuptools]
+packages = ["datacompy"]
+zip-safe = false
+include-package-data = true
+
+[tool.setuptools.package-data]
+"*" = ["templates/*.txt"]
+
+[tool.setuptools.dynamic]
+version = {attr = "datacompy.__version__"}
+
+[tool.distutils.bdist_wheel]
+python-tag = "py3"
+
+[project.optional-dependencies]
+duckdb = ["fugue[duckdb]"]
+polars = ["fugue[polars]"]
+spark = ["fugue[spark]"]
+dask = ["fugue[dask]"]
+ray = ["fugue[ray]"]
+docs = [
+	"sphinx",
+	"furo",
+	"myst-parser",
+]
+tests = [
+	"pytest",
+	"pytest-cov",
+	"pytest-spark",
+]
+qa = [
+	"pre-commit",
+	"black",
+	"isort",
+]
+build = [
+	"build",
+	"twine",
+	"wheel",
+]
+edgetest = [
+	"edgetest",
+	"edgetest-conda",
+]
```

Comment:
Inconsistent indentation with the rest of the file

Reply:
updated!